exercises/leap/canonical-data.json

```diff
@@ -1,24 +1,34 @@
 {
-   "cases": [
-      {
-         "description": "year not divisible by 4: common year",
-         "input": 2015,
-         "expected": false
-      },
-      {
-         "description": "year divisible by 4, not divisible by 100: leap year",
-         "input": 2016,
-         "expected": true
-      },
-      {
-         "description": "year divisible by 100, not divisible by 400: common year",
-         "input": 2100,
-         "expected": false
-      },
-      {
-         "description": "year divisible by 400: leap year",
-         "input": 2000,
-         "expected": true
-      }
-   ]
+  "exercise": "leap",
+  "version": "1.0.0",
+  "comments": [
+    "Given a Gregorian calendar year, indicate whether it is a leap year."
+  ],
+
+  "cases": [
+    {
+       "description": "year not divisible by 4: common year",
+       "property": "notLeap",
```

Comment:
#### About `isLeap` and `notLeap`

All the tests in this test suite use exactly the same test logic, only changing the test data, so they should have the same `property` value. `isLeapYear`and `leapYear` are good candidates